packages/angular-query-experimental/src/useIsFetching.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  DestroyRef,
+  assertInInjectionContext,
+  inject,
+  signal,
+} from '@angular/core'
+import {
+  type QueryClient,
+  type QueryFilters,
+  notifyManager,
+} from '@tanstack/query-core'
+import { QueryClientService } from './QueryClientService'
+import type { Signal } from '@angular/core'
+
+export function useIsFetching(
```

Comment:
Instead of the use keyword, in Angular, I'd use the inject keyword (injectIsFetching). 

This way the developer knows that this should be run in an injection context (because we are also using the inject fn inside this fn).

Reply:
Sounds like it might be a good idea to name these functions injectX, but I am not aware of such a naming convention. Do you maybe have some examples from open source software? If there are none it might still be a good idea but I would like to hear some more opinions on it.

Reply:
`use*` is a tsx convention.

There is no naming convention per say, but most of the Angular OSS is using `inject*` since the `inject` was introducted in v14. As @eneajaho said, it gives a warning to the developer that this function is injecting Services and need to be run in injection context.

Reply:
Open source Angular lib that uses inject convention a lot: https://ngxtension.netlify.app/utilities/injectors/inject-is-intersecting/

Reply:
While I agree that `inject` would be better, referencing your own library feels like cheating @eneajaho 😂

Reply:
[Angular Three](https://github.com/nartc/angular-three) is using a lot of them as well. but @eneajaho and Chau are from of the same species 🤣